### DISCUSSION TITLE
I give up.....

### DISCUSSION THREAD

**Discussion by smilingjackal** [NARUTOfzr/Neptune_3#59] [Q&A]
Been trying to update motherboard for 2 hrs. Someone tell me how to do this. Got V1-1.0.3 on printer. When I try to copy and load 4 or 5 screen on printer does nothing. Explain how like you're dealing with a child, or I'm lost.

**Comment by TheFeralEngineer** [comment]
format SD card to FAT32, 4096
copy ZNP_ROBIN_NANO.bin file to SD card
turn off printer
insert SD card
turn on printer
wait for firmware to finish updating
 
the important steps are to have a properly formatted card, a clean bin file and possibly update the screen firmware first.
